### PR TITLE
PA-470: Add libraries to create object locked s3 bucket

### PR DESCRIPTION
**Pull request by kshithijiyer-px**
**What this PR does / why we need it**:
- [x] Changing existing function to create object lock s3 bucket 

**Which issue(s) this PR fixes** (optional)
Closes #PA-470

**Special notes for your reviewer**:

### CODE REVIEW

File: tests/common.go [hunk 1]
```diff
@@ -3690,6 +3699,23 @@ func CreateS3Bucket(bucketName string) {
...
+		_, err = S3Client.PutObjectLockConfiguration(&s3.PutObjectLockConfigurationInput{
+			Bucket: aws.String(bucketName),
+			ObjectLockConfiguration: &s3.ObjectLockConfiguration{
+				ObjectLockEnabled: aws.String(enabled),
+				Rule: &s3.ObjectLockRule{
+					DefaultRetention: &s3.DefaultRetention{
+							Days: aws.Int64(retainCount),
+							Mode: aws.String(objectLockMode)}}}})
+
+		expect(err).NotTo(haveOccurred(),
```
Comment (lsrinivas-pure) [inline tests/common.go:3715]: use dash verification or return the error

File: tests/common.go [hunk 2]
```diff
@@ -3690,6 +3699,23 @@ func CreateS3Bucket(bucketName string) {
...
+		fmt.Sprintf("Failed to update Objectlock config with Retain Count [%v] and Mode [%v]. Error: [%v]", retainCount, objectLockMode, err))
```
Comment (lsrinivas-pure) [inline tests/common.go:3716]: Sprintf just formats the string, use log to print the log messsage

**Review by lsrinivas-pure — CHANGES_REQUESTED** [review]

File: tests/common.go (hunk 2 above)
Comment (kshithijiyer-px) [reply tests/common.go]: Done

File: tests/common.go (hunk 1 above)
Comment (kshithijiyer-px) [reply tests/common.go]: Done

**Review by lsrinivas-pure — APPROVED** [review]
LGTM

**Review by snigdha-px — APPROVED** [review]
LGTM

**Pull request merged**